rocketpool-cli/wallet/purge-keys.go

```diff
@@ -0,0 +1,44 @@
+package wallet
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+
+	"github.com/rocket-pool/smartnode/shared/services/rocketpool"
+	cliutils "github.com/rocket-pool/smartnode/shared/utils/cli"
+)
+
+func purgeKeys(c *cli.Context) error {
+
+	// Get RP client
+	rp, err := rocketpool.NewClientFromCtx(c)
+	if err != nil {
+		return err
+	}
+	defer rp.Close()
+
+	// Get & check wallet status
+	status, err := rp.WalletStatus()
+	if err != nil {
+		return err
+	}
+	if !status.WalletInitialized {
+		fmt.Println("The node wallet is not initialized.")
+		return nil
+	}
```

Comment:
I don't think this should be an early return-
If I initialize a wallet, then spin up a minipool, then run `rm .rocketpool/data/wallet`, then run `rocketpool wallet purge-keys`, my POLA senses would be triggered if smartnode decided not to delete the validator keys just because I manually deleted the node wallet:

```golang
// Check if the wallet has been initialized
func (w *Wallet) IsInitialized() bool {
        return (w.ws != nil && w.seed != nil && w.mk != nil)
}
```

Reply:
You're right. This command should delete validator keys under any condition. I'm removing this check.